src/gui/render_simple.h

```diff
@@ -45,8 +45,8 @@ static void conc4d(SCALERNAME,SBPP,DBPP,R)(const void *s) {
 #endif
 	/* Clear the complete line marker */
 	Bitu hadChange = 0;
-	const SRCTYPE *src = (SRCTYPE*)s;
-	SRCTYPE *cache = (SRCTYPE*)(render.scale.cacheRead);
+	auto src   = static_cast<const SRCTYPE*>(s);
+	auto cache = static_cast<SRCTYPE*>(static_cast<void*>(render.scale.cacheRead));
```

Comment:
Can `reinterpret_cast<SRCTYPE*>(render.scale.cacheRead);` avoid the double cast?

Reply:
OK, used `reinterpret_cast`, here and in 2 more places.